extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaMediaInsertDialog.js

```diff
@@ -151,7 +152,12 @@ ve.ui.WikiaMediaInsertDialog.prototype.initialize = function () {
 
 	this.mainPage = new OO.ui.PageLayout( 'main', { $content: this.$mainPage } );
 	this.searchPage = new OO.ui.PageLayout( 'search', { $content: this.search.$element } );
-	this.pages.addPages( [ this.mainPage, this.searchPage ] );
```

Comment:
Instead of renaming ```initialize``` method and having ```initializeWrapper``` take parameter would it make sense to create new method called ```addPages``` that default implementation be ```this.pages.addPages( [ this.mainPage, this.searchPage ] );``` and ```this.pages.addPages( [ this.searchPage ] );``` after overwriting?